src/main/createWindow.js

```diff
@@ -1,14 +1,27 @@
 import { app, BrowserWindow, dialog } from 'electron';
+import windowStateKeeper from 'electron-window-state';
+
 import setupBarMenu from './menu/setupBarMenu';
 import setupContextMenu from './menu/setupContextMenu';
 
 export default appState => {
+  // Load the previous state with fallback to defaults.
```

Comment:
is there any way that we can have a fully maximized default?

Reply:
I updated this PR so that if the window state is not saved yet (the default situation with new app installs), the display resolution is asked from Electron and that's used as the default.